ml_experiment/Scheduler.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, exp_name: str, seeds: list[int], entry: str, version: Version
         self.seeds = seeds
         self.entry = entry
         self.base_path = base or os.getcwd()
+        self.results_path = os.path.join(self.base_path, 'results', self.exp_name)
```

Comment:
I recently learned of `from pathlib import Path` which is part of the standard library: https://docs.python.org/3/library/pathlib.html

Let's you do some of this path manipulation stuff like this:
```python
self.results_path = Path(self.base_path) / 'results' / self.exp_name
```
which is pretty nice. Generally just simplifies filesystem related things a lot.

---

Anyways, no action item here (i.e. really no need to change this code), just thought I'd share!

Reply:
Oh cool!! I am always confused about which Path objects you can use as strings and which you cannot (PosixPath?). Thanks for the review!